src/array.test.js

```diff
@@ -73,4 +73,55 @@ describe("Test includes() function", () => {
     expect(testArr.myIncludes(3, -100)).toEqual(true)
     expect(testArr.myIncludes(1, -2)).toEqual(false)
   })
+})
+
+describe("Test Array.myMap()", () => {
+  it("Maps function on array", () => {
+    const inputArray = [1, 2, 3, 4]
+    const isEven = (index) => index % 2 == 0
+    const expectedArr = [false, true, false, true]
+    expect(inputArray.myMap(isEven)).toEqual(expectedArr)
+  })
+
+  it("Correctly passes index", () => {
+    const inputArray = [0, 0, 0, 0]
+    const expectedArr = [0, 1, 2, 3]
+    const toIndex = (_, i) => i
+    expect(inputArray.myMap(toIndex)).toEqual(expectedArr)
+  })
+
+})
+
+describe("Test Array.myReduce()", () => {
+  it("Reduces with no params", () => {
+    const inputArray = [1, 2, 3, 4];
+    const reducer = (accumulator, currentValue) => accumulator + currentValue;
+    expect(inputArray.myReduce(reducer)).toEqual(10)
+  })
+
+  it("Reduces with one param", () => {
+    const inputArray = [1, 2, 3, 4];
+    const reducer = (accumulator, currentValue) => accumulator + currentValue;
+    expect(inputArray.myReduce(reducer, 4)).toEqual(14)
+  })
+})
+
+describe("Test Array.lastIndexOf()", () => {
```

Comment:
These are calling the original array methods. Should be calling the method you wrote